htdocs/compta/bank/class/account.class.php

```diff
@@ -279,6 +279,11 @@ class Account extends CommonObject
 
 
 
+	/**
+	 * @var string Error message.
+	 */
+	public $error_message;
```

Comment:
We already have 
->error (for simple string) and ->errors (for an array of several error message)
After scan on ->->error_message it seems we use this at 4 places. i suggest instead to rename 
->error_message into ->error.